README.md

```diff
@@ -184,6 +184,10 @@ Then reproduce the bug and go to `Help > Show Log in Files` to open the log file
 
 ## References
 
+You still don't know why and what for you might need a Git Machete?
```

Comment:
Hmmm that doesn't sound well from PR PoV :c:c:c

Reply:
Maybe sth along the lines `for additional background on what we believe are good practices....`